packages/jazz-tools/src/implementation/ContextManager.ts

```diff
@@ -43,6 +43,10 @@ export class JazzContextManager<
   protected context: PlatformSpecificContext<Acc> | undefined;
   protected props: P | undefined;
   protected authSecretStorage = new AuthSecretStorage();
+  protected authSecretStorageWithNotify = Object.assign(
+    Object.create(this.authSecretStorage),
+    { notify: true },
+  );
```

Comment:
Decided to change the auth notify behavoir to be decoupled when used inside the context flow and to be automatic when used outside (e.g.  in the auth providers).

In the context flow we trigger an update at the same time when we trigger the context update, to align the auth state update timing with the account value updates.